Documentation/bpf/progtypes.rst

```diff
@@ -338,49 +338,55 @@ corresponding network driver name of an interface can be determined as follows:
     driver: nfp
     [...]
 
-+-------------------+------------+-------------+
-| Vendor            | Driver     | XDP Support |
-+===================+============+=============+
-| Amazon            | ena        | >= 5.6      |
-+-------------------+------------+-------------+
-| Broadcom          | bnxt_en    | >= 4.11     |
-+-------------------+------------+-------------+
-| Cavium            | thunderx   | >= 4.12     |
-+-------------------+------------+-------------+
-| Freescale         | dpaa2      | >= 5.0      |
-+-------------------+------------+-------------+
-| Intel             | ixgbe      | >= 4.12     |
-|                   +------------+-------------+
-|                   | ixgbevf    | >= 4.17     |
-|                   +------------+-------------+
-|                   | i40e       | >= 4.13     |
-|                   +------------+-------------+
-|                   | ice        | >= 5.5      |
-+-------------------+------------+-------------+
-| Marvell           | mvneta     | >= 5.5      |
-+-------------------+------------+-------------+
-| Mellanox          | mlx4       | >= 4.8      |
-|                   +------------+-------------+
-|                   | mlx5       | >= 4.9      |
-+-------------------+------------+-------------+
-| Microsoft         | hv_netvsc  | >= 5.6      |
-+-------------------+------------+-------------+
-| Netronome         | nfp        | >= 4.10     |
-+-------------------+------------+-------------+
-| Others            | virtio_net | >= 4.10     |
-|                   +------------+-------------+
-|                   | tun/tap    | >= 4.14     |
-|                   +------------+-------------+
-|                   | bond       | >= 5.15     |
-+-------------------+------------+-------------+
-| Qlogic            | qede       | >= 4.10     |
-+-------------------+------------+-------------+
-| Socionext         | netsec     | >= 5.3      |
-+-------------------+------------+-------------+
-| Solarflare        | sfc        | >= 5.5      |
-+-------------------+------------+-------------+
-| Texas Instruments | cpsw       | >= 5.3      |
-+-------------------+------------+-------------+
++-------------------+---------------+-------------+
+| Vendor            | Driver        | XDP Support |
++===================+===============+=============+
+| Amazon            | ena           | >= 5.6      |
++-------------------+---------------+-------------+
+| Broadcom          | bnxt_en       | >= 4.11     |
++-------------------+---------------+-------------+
+| Cavium            | qed           | >= 4.10     |
```

Comment:
Where does this one come from?

Reply:
QLogic (Cavium) qed* drivers | 4.10 | 496e05170958
-- | -- | --

Reply:
> QLogic is Cavium's acquisition.

Ah right now I see, thanks Maxim.

But we already have an entry for QLogic, so adding Cavium creates a duplicate.

> I wonder if we should rename Mellanox to NVIDIA, or stick with QLogic, or provide both names in both cases.

Good question. I'd stick with the current names, that are also easier to map to kernel drivers, and change if the names change in the kernel tree?